### ISSUE TITLE
Missing Category warnings

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#2404]
To resurrect the tabs has some unwanted side effects. The console is now flooded with missing category warnings. We should do something here.

Either we supress the warning. Or we update all the panel classes ...

![warnings](https://user-images.githubusercontent.com/13919702/117564087-7e033380-b0aa-11eb-9f6d-f290a09fb3c3.jpg)

**Comment by ReinerBforartists** [comment]
Oddly enough, this happens after the merge. The build before the merge works fine. So something has changed it seems.

**Comment by ReinerBforartists** [comment]
Ah, this warnings just shows with a debug build.

**Comment by iyadahmed** [comment]
yep not a show stopper, panels without category goes into "Misc" category, it should be fixed though
![image](https://user-images.githubusercontent.com/25552173/117585203-775bd700-b111-11eb-94ba-71955839ecda.png)
